test/tools/WebListener/README.md

```diff
@@ -59,6 +59,30 @@ Response when certificate is not provided in request:
 }
 ```
 
+## /Delay/
+
+Returns the same results as the Get test. If a number is supplied, the server will wait that many seconds before returning a response. This can be used to test timeouts.
+
+```powershell
+Invoke-WebRequest -Uri 'http://localhost:8083/Delay/5'
```

Comment:
Most of the tests are using the Get-WebListenerUrl function. Suggest adding an example for that as well.

$uri = Get-WebListenerUrl -Test 'Delay' -TestValue '5'

Reply:
I will add those on the next iteration. Created #4918 to rack.